### PR TITLE
SAK-44975 - Lessons > Add Question > Short Answer > Grading Should No…

### PR DESCRIPTION
**Pull request by mpellicer**
…t Require a Possible Answer

### CODE REVIEW

File: lessonbuilder/tool/src/webapp/js/show-page.js [hunk 1]
```diff
@@ -3650,13 +3650,9 @@ function prepareQuestionDialog() {
 	    $('#question-error').text(msg("simplepage.question-need-2"));
 	    $('#question-error-container').show();
 	    return false;
-	} else if ($("#shortanswerSelect").prop("checked") && $("#question-graded").prop("checked") &&
-		   $(".question-shortanswer-answer").filter(function(index){return $(this).val()!=="";}).length < 1) {
-	    $('#question-error').text(msg("simplepage.question-need-1"));
```
Comment (bgarciaentornos) [inline lessonbuilder/tool/src/webapp/js/show-page.js:3655]: Do you think we could also remove the translation keys for that message?

Comment (mpellicer) [reply lessonbuilder/tool/src/webapp/js/show-page.js:3655]: We can but not sure people want to remove the restriction.

Comment (bgarciaentornos) [reply lessonbuilder/tool/src/webapp/js/show-page.js:3655]: Ok, but if we remove it then we should get rid of the key.

**Review by bgarciaentornos — CHANGES_REQUESTED** [review]

File: lessonbuilder/tool/src/webapp/js/show-page.js (hunk 1 above)
Comment (mpellicer) [reply lessonbuilder/tool/src/webapp/js/show-page.js:3655]: Okay!

**Review by bgarciaentornos — APPROVED** [review]

**Pull request merged**